win/libsass.sln

```diff
@@ -1,10 +1,21 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30723.0
+# Visual Studio 14
```

Comment:
I'm pretty sure you're correct, since it's only the project toolsversion stuff that breaks backwards compat. Maybe AppVeyor should be updated too (separately) https://github.com/sass/libsass/blob/2847edc45d1f887277fba5cd6873435e50da6f9b/appveyor.yml#L1